test/strategies/segment_envelope.cpp

```diff
@@ -1,6 +1,6 @@
 // Boost.Geometry
```

Comment:
now there is strategy named `envelope_segment` the name of this file should also change

Reply:
Will do.